Tests/POSIXTests/EnvTests.swift

```diff
@@ -11,8 +11,13 @@
 import XCTest
 
 import POSIX
+import TestSupport
```

Comment:
I don't know if the tests for things in `TestSupport` should go here or in its own module perhaps.
@ddunbar thoughts?